platform_versioned_docs/version-23.4.0/functionality_matrix/functionality_matrix.mdx

```diff
@@ -22,11 +23,13 @@ nf-launcher versions prefixed with j17 refer to Java version 17; j11 refers to J
 | 23.3.0           | j17-23.04.3         | 23.04.3          | 2.1            |
 | 23.2.0           | j17.23.04.2-up3     | 23.04.2          | 2.1            |
 | 23.1.3           | j17-23.04.1         | 23.04.1          | 2.1            |
-| 22.4.1           | j17-22.10.6         | 22.10.6          | -              |
-| 22.4.0           | j17-22.10.6         | 22.10.6          | -              |
+
+(* - Nextflow version 24.04.2 will be the baseline version of an upcoming Platform release, but you can take advantage of [new Nextflow features](https://nextflow.io/blog/2024/nextflow-2404-highlights.html) in current Platform versions by specifying `NXF_VER=24.04.2` in your configuration.)
 
 ---
 
+To use a Nextflow version other than the baseline in your pipeline run, include the environment variable `NXF_VER` in a [pre-run script](./launch/advanced#pre-and-post-run-scripts) during launch. For example, `NXF_VER=24.04.2`. 
```

Comment:
Do we need to call out that post 24.04.x edge versions use Fusion v2.4 by default?